packages/patternfly-3/react-tree-view/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react-tree-view",
```

Comment:
this npm package name is already taken it seems... maybe something like `patternfly-react-wooden-tree` fits better?
https://www.npmjs.com/package/react-tree-view

Reply:
Ok, should I rename the folder as well?

Reply:
Once you've chosen a package name, I think the only thing we really need will be to add a line to the README providing instructions on how to install the module.  Something like

### Getting Started
```sh
npm install <package_name> --save
```